uni-resolver-web/src/main/resources/application.yml

```diff
@@ -297,3 +297,7 @@ uniresolver:
       propertiesEndpoint: 'true'
       testIdentifiers:
         - did:polygonid:polygon:mumbai:2qDj9EDytmvtQP1or3FxykXGEaqSA1ss479MYHDMJc
+    - pattern: "^(did:pdc:.+)$"
+        url: http://driver-did-pdc:8080/
```

Comment:
Pls fix the indentation.  
Atm this is breaking the uni-resolver application

Reply:
Sorry, I was careless. I have revised and committed it.